exporter/datadogexporter/config/config.go

```diff
@@ -54,10 +54,6 @@ func (api *APIConfig) GetCensoredKey() string {
 
 // MetricsConfig defines the metrics exporter specific configuration options
 type MetricsConfig struct {
-	// Namespace is the namespace under which the metrics are sent
-	// By default metrics are not namespaced
-	Namespace string `mapstructure:"namespace"`
```

Comment:
What happens with people already using this? I know we are still allowed to make breaking changes, but establishing a procedure for handling this kind of scenario is what 0.x is for as well ;-)

Reply:
This component has not yet been in a 0.x release (it should be on the next one), I think a breaking change of this kind on a nightly is acceptable

Reply:
You can keep the field in the API (config, in this case), documented as "deprecated", logging a warn in case it's used. You don't need to use its value if you are just dropping the property entirely. On the next possible release (or n+2, preferably) , you remove the field. This way, people have a chance of getting notified without reading the release notes and have a chance of scheduling this update on their end.

Reply:
Makes sense, thank you. As I said, since this was not enabled until fairly recently I think it's safe to remove it without deprecation (so long as this PR gets merged before 0.14), but we will take that procedure into account for future breaking changes